packages/components/src/Actions/ActionButton/Button.stories.js

```diff
@@ -158,25 +158,3 @@ export const Default = () => (
 		</div>
 	</div>
 );
-
-export const Combinations = withPropsCombinations(ActionButton, {
```

Comment:
why do we remove this story?

Reply:
Because this uses react-storybook-addon-props-combinations which is no longer maintained.

As teams should use now the action provided by the DS, no need to make an effort to maintain this story